src/DebugApi.ts

```diff
@@ -54,7 +55,7 @@ export class DebugApi {
                 }
                 log.error(err.stack);
             }
-        }).listen(this.port);
+        }).listen(this.port, this.host);
```

Comment:
[The official docs](https://nodejs.org/api/net.html#serverlistenport-host-backlog-callback) states that omitting this value has the behaviour of picking IPv6 if available and falling back to IPv4. 
I think it's important that this can be `undefined`.